sql/aiven_extras.sql

```diff
@@ -171,7 +171,8 @@ $$;
 
 DROP FUNCTION IF EXISTS aiven_extras.pg_drop_subscription(TEXT);
```

Comment:
I would expect that the next version now also needs a 

```sql
DROP FUNCTION IF EXISTS aiven_extras.pg_drop_subscription(TEXT, BOOLEAN);
```

Might be nice to already add it now?

Reply:
Thanks!